packs/spells/etheric-shards.json

```diff
@@ -37,46 +37,17 @@
             "value": "1 minute"
         },
         "heightening": {
+            "area": "5",
             "damage": {
-                "u4xykvzyx1ej6xl4": "+1"
+                "u4xykvzyx1ej6xl4": "1"
             },
             "interval": 2,
             "type": "interval"
         },
         "level": {
             "value": 5
         },
-        "overlays": {
-            "cZnJKUusAPSiR1de": {
-                "_id": "cZnJKUusAPSiR1de",
-                "name": "Etheric Shards (5)",
-                "overlayType": "override",
-                "sort": 1,
-                "system": {}
-            },
-            "i6bpmbq7m2xyhw5h": {
-                "_id": "i6bpmbq7m2xyhw5h",
-                "name": "Etheric Shards (7)",
-                "overlayType": "override",
-                "sort": 2,
-                "system": {
-                    "area": {
-                        "value": 25
-                    }
-                }
-            },
-            "qc6zredmnvsz6x0m": {
-                "_id": "qc6zredmnvsz6x0m",
-                "name": "Etheric Shards (9)",
-                "overlayType": "override",
-                "sort": 3,
-                "system": {
-                    "area": {
-                        "value": 30
-                    }
-                }
-            }
-        },
+        "overlays": {},
```

Comment:
```suggestion
```

Reply:
Done, though I'll note that this is just how it extracted.